build/build.go

```diff
@@ -316,13 +316,6 @@ func parseAndAugment(bctx *build.Context, pkg *build.Package, isTest bool, fileS
 		},
 	}
 
-	// reflect needs to tell Go 1.11 apart from Go 1.11.1 for https://github.com/gopherjs/gopherjs/issues/862,
-	// so provide it with the custom go1.11.1 build tag whenever we're on Go 1.11.1 or later.
-	// TODO: Remove this ad hoc special behavior in GopherJS 1.12.
-	if runtime.Version() != "go1.11" {
-		nativesContext.ReleaseTags = append(nativesContext.ReleaseTags, "go1.11.1")
-	}
-
```

Comment:
As `runtime.Version() != "go1.11"` becomes always true, `nativesContext.ReleaseTags = append(nativesContext.ReleaseTags, "go1.11.1")` should always be kept. Is this correct?

Reply:
It should not be kept, because it's no longer needed. This is described in the commit PR description and my comment above.

Reply:
OK I understood this was originally not needed.